contracts/lib/OrderCombiner.sol

```diff
@@ -250,8 +250,11 @@ contract OrderCombiner is OrderFulfiller, FulfillmentApplier {
                 // Retrieve array of offer items for the order in question.
                 OfferItem[] memory offer = advancedOrder.parameters.offer;
 
+                // Read length of offer array and place on the stack.
+                uint256 totalOffers = offer.length;
```

Comment:
Nit: the offer is an array of offer items, so there's no plural of it — rename to `totalOfferItems`

Reply:
ah got it that makes sense. let me update. thanks.

btw the stack-too-deep errors should be better handled in 0.8.14 right? so is it no longer an issue?

Reply:
can't find a source at the moment so disregard. will tag you on this PR with a source if/when i find it.